src/components/IconLabel/types.ts

```diff
@@ -21,6 +21,7 @@ interface PeriodProps {
 
 interface RoundPeriodProps {
   type: "roundPeriod";
+  label: string;
```

Comment:
```suggestion
  label?: string;
```
let's put it optional for retro compatibility